src/org/opendatakit/briefcase/ui/SettingsPanel.java

```diff
@@ -24,83 +22,97 @@
+  private final JCheckBox chkParallel = new JCheckBox(MessageStrings.PARALLEL_PULLS);
+  private final JCheckBox chkTrackingConsent = new JCheckBox(MessageStrings.TRACKING_CONSENT);
+  private final JCheckBox chkStorePasswordsConsent = new JCheckBox("Store passwords (unsafe)");
+
+  SettingsPanel(final MainBriefcaseWindow parentWindow) {
+    txtBriefcaseDir.setFocusable(false);
+    txtBriefcaseDir.setEditable(false);
+    txtBriefcaseDir.setColumns(50);
+
+    final JButton btnChoose = new JButton("Change...");
+    btnChoose.addActionListener(new ActionListener() {
+      @Override
+      public void actionPerformed(ActionEvent arg0) {
+        WrappedFileChooser fc = new WrappedFileChooser(parentWindow.frame,
+            new BriefcaseFolderChooser(parentWindow.frame));
+        // figure out the initial directory path...
+        String candidateDir = txtBriefcaseDir.getText();
+        File base = null;
+        if (candidateDir == null || candidateDir.trim().length() == 0) {
+          // nothing -- use default
+          base = new File(BriefcasePreferences.appScoped().getBriefcaseDirectoryOrUserHome());
+        } else {
+          // start with candidate parent and move up the tree until we have a valid directory.
+          base = new File(candidateDir).getParentFile();
+          while (base != null && (!base.exists() || !base.isDirectory())) {
+            base = base.getParentFile();
+          }
+        }
+        if (base != null) {
+          fc.setSelectedFile(base);
+        }
+        int retVal = fc.showDialog();
+        if (retVal == JFileChooser.APPROVE_OPTION) {
+          File parentFolder = fc.getSelectedFile();
+          if (parentFolder != null) {
+            String briefcasePath = parentFolder.getAbsolutePath();
+            txtBriefcaseDir.setText(briefcasePath);
+            BriefcasePreferences.setBriefcaseDirectoryProperty(briefcasePath);
+            FileSystemUtils.createFormCacheInBriefcaseFolder();
+            parentWindow.storageLocation.establishBriefcaseStorageLocation(parentWindow.frame, parentWindow);
+          }
+        }
+      }
+    });
+
+    ProxyChangeListener proxyChangeListener = new ProxyChangeListener();
+
+    txtHost.setEnabled(false);
+    txtHost.addFocusListener(proxyChangeListener);
+
+    spinPort.setEnabled(false);
+    spinPort.addChangeListener(proxyChangeListener);
+
+    chkProxy.setSelected(false);
+    chkProxy.addActionListener(new ProxyToggleListener());
+
+    chkParallel.setSelected(BriefcasePreferences.getBriefcaseParallelPullsProperty());
+    chkParallel.addActionListener(new ParallelPullToggleListener());
+
+    chkTrackingConsent.setSelected(BriefcasePreferences.getBriefcaseTrackingConsentProperty());
+    chkTrackingConsent.addActionListener(new TrackingConsentToggleListener(parentWindow.briefcaseAnalytics));
+
+    chkStorePasswordsConsent.setSelected(BriefcasePreferences.getStorePasswordsConsentProperty());
```

Comment:
Maybe add a tooltip text? 
`chkStorePasswordsConsent.setToolTipText("Passwords are saved in plaintext in the local file system");`
I don’t know if we have a policy on using them.